### PR TITLE
Ensure args are JSON-compatible

### PR DESCRIPTION
**Pull request by andhapp**
Fixes Sidekiq deprecation warning, that would have been a problem in Sidekiq 7.0

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -4,6 +4,15 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+## [1.1.3] - 2021-05-15
```
Comment (zorbash) [inline CHANGELOG]: Let's make this 2022. Time flies! 😃

**Review by zorbash — APPROVED** [review]

File: CHANGELOG (hunk 1 above)
Comment (andhapp) [reply CHANGELOG]: Done 🤣

**Comment by andhapp** [comment]
@zorbash Can we get it released, please? Thanks!

**Comment by zorbash** [comment]
@andhapp Released.

**Pull request merged**